hazyCalculator.js

```diff
@@ -3,16 +3,17 @@ function isSkippedValue(value) {
 }
```

Comment:
This will treat `null` as a skipped character when it should be treated as numeric. This should have a `&& value !== null` as well.